net-misc/bird/bird-2.0.10.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools fcaps
+
+DESCRIPTION="A routing daemon implementing OSPF, RIPv2 & BGP for IPv4 & IPv6"
+HOMEPAGE="https://bird.network.cz"
+SRC_URI="ftp://bird.network.cz/pub/${PN}/${P}.tar.gz"
+LICENSE="GPL-2"
```

Comment:
```suggestion
LICENSE="GPL-2+"
```

Reply:
It’s weird that pkgdev didn’t had it automatically (it worked with repoman), but I force-pushed with it anyway.